src/js/components/Select/README.md

```diff
@@ -334,11 +334,10 @@ boolean
 
 **placeholder**
 
-Placeholder text to use when no value is provided.
+Placeholder to use when no value is provided.
 
 ```
 string
```

Comment:
thanks for cleaning up the node type, let's run TS tests on select to make sure there are no regressions